src/locale/languages/en.js

```diff
@@ -248,6 +248,12 @@ export const en = {
       areYouSurePrompt: 'Are you sure?',
       applyLabelTitle: 'Apply a label to this issue',
     },
+    comment: {
```

Comment:
Please duplicate these all new keys into the remaining language files.

Reply:
Should I try to translate? or I add the text in English?

Reply:
Generally not, although if you know any language and can translate, it would be good, but if not, then it is enough to duplicate it in English, otherwise there will be errors (because new keys will not be in the remaining languages)